src/NumberGuessingGame.jsx

```diff
@@ -2,76 +2,63 @@ import React, { Component } from "react";
 import GuessControl from "./GuessControl";
 import GuessMessage from "./GuessMessage";
 import GameOver from "./GameOver";
+import { useState } from "react";
+
 
-/**
- *
- * Returns a random integer number from 1-100 inclusive
- */
 function getRandomNumber() {
   return Math.floor(Math.random() * 100) + 1;
 }
 
 const MAX_ATTEMPTS = 5;
 
-class NumberGuessingGame extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    };
-
-    /**
-     * These lines are required to make the methods/functions declared on this
-     *  class have the correct `this` object when they run.
-     */
-    this.handleGuess = this.handleGuess.bind(this);
-    this.handleReset = this.handleReset.bind(this);
-  }
-
-  handleGuess(guess) {
-    this.setState({
-      latestGuess: guess,
-      numberOfGuesses: this.state.numberOfGuesses + 1,
-    });
-  }
-
-  handleReset() {
-    this.setState({
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    });
-  }
-
-  render() {
-    const isCorrectGuess = this.state.latestGuess === this.state.numberToGuess;
+const NumberGuessingGame = () => {
+
+  
+
+  const [numberToGuess, setNumberToGuess] = useState(Math.floor(Math.random() * 100) + 1);
```

Comment:
There is a `getRandomNumber` function which you have above why not just call the function as the value of the state instead of repeating yourself 

```suggestion
  const [numberToGuess, setNumberToGuess] = useState(getRandomNumber())
```